### PR TITLE
fix: Azure Ingestion changes

### PR DESCRIPTION
**Pull request by rramalinga**
## Description

Gaia changes for Azure Ingestion objects.
New object - PublicIP Address being introduced
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "backend",
    "pr-id": "1545",
    "commit-sha": "8675d573cdc0af760000736857f49c2085585e5c",
    "pipeline": "master"
  },
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "gaia",
    "pr-id": "765",
    "commit-sha": "60cd2e133ae3a0449784790dfe34b3c65451b698",
    "pipeline": "master"
  }
]
```

File: custom_validations.go [hunk 1]
```diff
@@ -1449,7 +1449,7 @@ func ValidateCloudTag(attribute string, tag string) error {
 }
 
 // nativeIDRegex is the regular expression to check the format of the nativeID.
-var nativeIDRegex = regexp.MustCompile(`^[a-zA-Z0-9-_#+.\/:@]+$`)
+var nativeIDRegex = regexp.MustCompile(`^[a-zA-Z0-9-_#+.\/:@{}"\\,]+$`)
```
Comment (malkhamis-panw) [inline custom_validations.go:1452]: let's update `TestValidateNativeID` to cover these changes to avoid future regressions

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "backend",
    "pr-id": "1545",
    "commit-sha": "75221e5431083b6bedb43c80585318dea33049e6",
    "pipeline": "master"
  },
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "gaia",
    "pr-id": "765",
    "commit-sha": "107d2975dc68c005da10fa09be726f4d2f76032c",
    "pipeline": "master"
  }
]
```

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "backend",
    "pr-id": "1545",
    "commit-sha": "705e38054d5b2a8b0dd300a5c22dab584db60ca1",
    "pipeline": "master"
  },
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "gaia",
    "pr-id": "765",
    "commit-sha": "107d2975dc68c005da10fa09be726f4d2f76032c",
    "pipeline": "master"
  }
]
```

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "backend",
    "pr-id": "1545",
    "commit-sha": "7ffdc4a62bd126125c4084af36567f37742bca36",
    "pipeline": "master"
  },
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "gaia",
    "pr-id": "765",
    "commit-sha": "107d2975dc68c005da10fa09be726f4d2f76032c",
    "pipeline": "master"
  }
]
```

**Review by malkhamis-panw — APPROVED** [review]

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "backend",
    "pr-id": "1545",
    "commit-sha": "d2523ff78bd69afde1fc084374ff72e7fdc64ff2",
    "pipeline": "master"
  },
  {
    "project": "fix-pandemona-azure-ingestion",
    "component": "gaia",
    "pr-id": "765",
    "commit-sha": "b81022ac6eec1f6b3cf156d65822bc5dac8da3c1",
    "pipeline": "master"
  }
]
```

**Pull request merged**